### ISSUE TITLE
The integration doesn't work for me

### ISSUE THREAD

**Issue by Valdi-1994** [zaubererty/homeassistant-4heat#7]
I have installed the integration but it doesn't give me any more options than the ones I show in the screenshot.
Is it possible that the installation was not correct? In the smartphone app it works perfectly, but in HA it doesn't work well.
Can anyone help me?

![image](https://github.com/user-attachments/assets/d4d1b3bd-9b43-480c-87e7-9d2b5d44a5ba)
![image](https://github.com/user-attachments/assets/b36988e2-2c6d-4fda-b720-6ca8e174a613)

 - home-assistant [Home Assistant OS 14.0]
 - Charger Firmware Version: [0.6.0]

**Comment by Stanojoski** [comment]
That is weird.. i have 12 entities.  did you try to reinstall it? 

![image](https://github.com/user-attachments/assets/6f9d5b15-6539-427a-829e-23f360e9bba7)

**Comment by Valdi-1994** [comment]
yes! I reinstall it but doesn´t works. is it possible that i didn´t install it correctly¿ could some tell me how tominstall_

**Comment by toccata10** [comment]
I have exactly the same problem with just 2 entities. Could it be that Hacs has deprecated the sensor in their new versions ?

**Comment by xtrapowa** [comment]
Hi, same problem for me. Deprecated sensors was corrected in my config files but still same with just 2 entities too.

**Comment by Valdi-1994** [comment]
Is anyone else having this problem? I bought the wifi module to integrate with HA and it doesn't work for me... it's a shame

**Comment by Stanojoski** [comment]
It still works fine for me 
Please check the https://community.home-assistant.io/t/tiemme-4heat-stove-wifi-controller/374040 forum, where the development began. There are some useful tips for determining what is wrong. 

![image](https://github.com/user-attachments/assets/ec10bb14-5e0a-4dce-8b9b-91094cad191a)

![image](https://github.com/user-attachments/assets/413e2d8c-3bb2-4a47-b776-a53714010944)

**Comment by Stanojoski** [comment]
hello all,

Today it stopped working at my HA also..
There is a fix at https://github.com/zaubererty/homeassistant-4heat/issues/8 from Carloda that works. 
Can you try that?

**Comment by xtrapowa** [comment]
Hi all, 

Still some problems with the 3 errors below :( 

Detected code that calls async_forward_entry_setup for integration 4heat with title: Stove and entry_id: 01JGRZJ6JR1Y39KGFP2SSTFCNH, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1, please report this issue

<img width="722" alt="image" src="https://github.com/user-attachments/assets/0bb2ff8b-b824-4bfc-aabc-2706160f9ef1" />

**Comment by carloda** [comment]
> Hi all,
> 
> Still some problems with the 3 errors below :(
> 
> Detected code that calls async_forward_entry_setup for integration 4heat with title: Stove and entry_id: 01JGRZJ6JR1Y39KGFP2SSTFCNH, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1, please report this issue
> 
> <img alt="image" width="722" src="https://private-user-images.githubusercontent.com/45213025/400150441-0bb2ff8b-b824-4bfc-aabc-2706160f9ef1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwMTIyOTQsIm5iZiI6MTczNjAxMTk5NCwicGF0aCI6Ii80NTIxMzAyNS80MDAxNTA0NDEtMGJiMmZmOGItYjgyNC00YmZjLWFhYmMtMjcwNjE2MGY5ZWYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA0VDE3MzMxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwMDQ2OTRiMjkyZmExZDAwMjA1YWJjZmFhZmVlMWE0MGMwNjQyOWEyMTkwMWM1ZjcwNmE2NDNiY2NhY2VmNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xkIdcbHWawWdLPhj4kGGxFc-7re_yG_dzgUzYwkvlKU">

What version of Home Assistant do you have? Have you applied the changes I posted?

**Comment by carloda** [comment]
@xtrapowa  Try this, replace it with yours and reboot HA completely https://file.io/qwbSfe1XGUUs

**Comment by xtrapowa** [comment]
I tried with all your files and still the same : 

<img width="797" alt="image" src="https://github.com/user-attachments/assets/26346208-bbbb-43a3-a46f-5eefdd52b8a2" />

**Comment by carloda** [comment]
@xtrapowa 

Try with this procedure:

1 – Remove the integration
![Screenshot 2025-01-05 172734](https://github.com/user-attachments/assets/d8a7fb59-820f-4757-8e1e-257a30746015)

2 – Delete the “4heat” folder located in “custom_components”
3 – Fully restart HA (NO quick restart)
4 – After restarting, copy the “4heat” folder back into “custom_components” using the version I shared with you earlier (https://drive.google.com/file/d/1ozSGgmbZ5R744ttKS_lFbIVn2ouzSBxL/view?usp=sharing)
5 – Fully restart HA again (NO quick restart)
6 – Once restarted, reconfigure the integration.

Let me know.

**Comment by xtrapowa** [comment]
@carloda ,
Hi,

still the same : (

<img width="720" alt="image" src="https://github.com/user-attachments/assets/7c1a1806-7472-4c12-8cb2-74a706fb189c" />
